contentScript.js

```diff
@@ -0,0 +1,6 @@
+var s = document.createElement('script');
+s.src = chrome.extension.getURL('courses.js');
+s.onload = function() {
+    this.remove();
+};
+(document.head || document.documentElement).appendChild(s);
```

Comment:
Is it necessary to explicitly append our content script to the DOM like this each time? I would guess that there has to be an easier way to do this, since this seems a bit roundabout, but I could be wrong.

Actually, looking through the manifest, it seems that this is never called, so perhaps it is unnecessary.